snapcraft/tests/test_plugin_rust.py

```diff
@@ -0,0 +1,78 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from unittest import mock
+
+import snapcraft
+from snapcraft import tests
+from snapcraft.plugins import rust
+
+
+class MakePluginTestCase(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        class Options:
+            makefile = None
+            make_parameters = []
+
+        self.options = Options()
+        self.project_options = snapcraft.ProjectOptions()
+
+    def test_schema(self):
+        schema = rust.RustPlugin.schema()
+
+        properties = schema['properties']
+        self.assertTrue('rust-channel' in properties,
+                        'Expected "rust-verion" to be included in properties')
```

Comment:
Looks like there are some typos here and in the `self.assertTrue` statements below: `verion` where I presume `version` is usually meant ... ?

Although in the specific line above, it looks like it should be `Expected "rust-channel"`